rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -336,6 +337,12 @@ upgrade_ncf() {
   fi
 }
 
+# Call rudderify on all local ncf techniques to make sure the promises will be properly generated
+rudderify_techniques() {
+  STEP="Call rudderify on all local ncf techniques (#7443)"
+  /usr/share/ncf/tools/ncf_rudder.py rudderify_technique /var/rudder/configuration-repository/techniques/ncf_techniques
```

Comment:
Fonction called should be rudderify_techniques !

Rudderify_technique only rudderify the technique passes as parameter!!